arch/arm64/core/mmu.c

```diff
@@ -804,7 +804,7 @@ static void enable_mmu_el1(struct arm_mmu_ptables *ptables, unsigned int flags)
 	isb();
 
 	/* Invalidate all data caches before enable them */
-	sys_cache_data_all(K_CACHE_INVD);
```

Comment:
Just want to note that we are affected by this change while working on internal firmware upgrade from v2.x, hopefully we are the only out-of-tree user..